.drone.yml

```diff
@@ -234,6 +234,27 @@ environment:
 
 steps:
 
+- name: replay-test
+  image: simplestakingcom/tezedge-ci-builder:latest
+  user: root
+  pull: if-not-exists
+  volumes:
+    - name: build
+      path: /artifacts
+    - name: tezedge-snapshots
+      path: /tmp/tezedge-snapshots
+    - name: tezedge-node-snapshotted-data
+      path: /tmp/tezedge_developer
+  commands:
+    - rm -rf /tmp/tezedge_developer/data
+    - unzip /tmp/tezedge-snapshots/tezedge_data_from_block_0_to_26907.zip # TODO: change this name (typo 26907 -> 27907)
+    - mv usr/local/etc/tezedge-data/tezedge_27708_2021-05-26T12:45:48.076277293+00:00/_data /tmp/tezedge_developer/data
+    - rust_libs=$(echo "`rustup show home`/toolchains/`rustup show active-toolchain | tr " " "\n" | head -1`/lib")
+    - export LD_LIBRARY_PATH="$${BUILD_ARTIFACTS_PATH}/build_files/ffi:$rust_libs"
+    - mkdir -p target/release
```

Comment:
@sebastiencs for our future selves (or anyone else who may come later to touch this code here) please leave a note about why we need to copy the protocol-runner executable there

Reply:
@tizoc Ok

Reply:
@tizoc It's done. The PR is ready to be merged